pkg/autoscaler/autoscaler/pdplan_autoscaler.go

```diff
@@ -0,0 +1,93 @@
+// Copyright 2020 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package autoscaler
+
+import (
+	"github.com/pingcap/tidb-operator/pkg/apis/pingcap/v1alpha1"
+	"github.com/pingcap/tidb-operator/pkg/label"
+	"github.com/pingcap/tidb-operator/pkg/pdapi"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/selection"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+const groupLabelKey = "group"
+
+func (am *autoScalerManager) syncPlans(tc *v1alpha1.TidbCluster, tac *v1alpha1.TidbClusterAutoScaler, plans []pdapi.Plan) error {
+	planGroups := sets.String{}
+	groupPlanMap := make(map[string]pdapi.Plan)
+	for _, plan := range plans {
+		groupName := plan.Labels[groupLabelKey]
+		planGroups.Insert(groupName)
+		groupPlanMap[groupName] = plan
```

Comment:
There is once an issue that we have discussed which needs PD to support:
For the same metric and the same component, if only one resrouce_type is specified and then users updated its resource_type, in the following responses, it will include both of the groups.
So, in this case, the gourpName may need to including the resources info?

Reply:
Identifying the configuration for the same `resource_type` during the instances in groups is hard to maintain in PD and PD won't support this feature. Currently, I think we can `ignore situtation if resource_type configuraiton changed` because that is not our main target and it doesn't block our main process. 

To solve this problem, I think there are many ways, and we can discuss it later, but not now.

Reply:
so what's the behavior for this issue now?

Reply:
And we ever had a deal during the design review @rleungx

Reply:
OK, discussed this with @rleungx, PD only supports one resource type now.